mlocal/frags/Makefile.stub

```diff
@@ -29,18 +29,21 @@ go_BIN := $(singularity)
 go_OBJ := $(SOURCEDIR)/src/pkg/buildcfg/config.go
 go_INSTALL := $(singularity_INSTALL)
 
+bash_completion :=  $(SOURCEDIR)/etc/bash_completion.d/singularity
+bash_completion_INSTALL := /etc/bash_completion.d/singularity
```

Comment:
Should this be installed at the system `/etc/bash_completion.d` or at `$(PREFIX)/etc/bash_completion.d` ??  @yhcote

Reply:
Well, I don't think bash will look into /usr/local, first impression is that bash expects its config from a tree off /etc/bash_completion.d/, but I might be wrong.

Reply:
is `$(PREFIX)/etc/bash_completion.d` sourced at boot?

Reply:
lol I was asking the same, for a matter of secs

Reply:
I ask because even with autoconf, if you just define `--prefix` then _everything_ should be installed under that prefix. You'd need to do something like:

`./configure --prefix=/usr/local --sysconfdir=/etc` 

To get the behavior you have going here.

Reply:
from the documentation:
 - https://debian-administration.org/article/316/An_introduction_to_bash_completion_part_1

Reply:
From my system `$HOME/.bash_profile`

```
    . /etc/bash_completion.d/git-completion.bash;
  [ ... ]
    . /usr/local/singularity/2.6.0/etc/bash_completion.d/singularity
```

So uhh... I'm sourcing each specific file I want. I do have a global profile script that sets up the generic completion helpers... but beyond that, none of the completion files get sourced automatically. I have to manually source them if I want them.

Reply:
**NOTE** : if this change is done, the how to install the `bash_completion` file will have to be detailed on the install documentation, so users can setup sourcing the file on their `$HOME/.bash_profile` script